src/fs.ts

```diff
@@ -11,7 +12,7 @@ export function getPackageJSON(cwd = process.cwd()): any {
       return data
     }
     catch (e) {
-      console.warn('Failed to parse package.json')
+      !options.silent ? console.warn('Failed to parse package.json') : 0
       process.exit(0)
```

Comment:
Unrelated to this PR, but I wondered is the exit code was intentionally made `0` instead of `1`?